app/Http/Controllers/CountryController.php

```diff
@@ -9,7 +9,7 @@ class CountryController extends Controller
     public function index()
     {
         // TASK: load the relationship average of team size
-        $countries = Country::all();
+        $countries = Country::avg(Team::class)->get();
```

Comment:
This won't work. What you wanna use is the [aggregate function](https://laravel.com/docs/8.x/eloquent-relationships#other-aggregate-functions) `withAvg`. That takes the name of the function you defined for the relationship and the column you wanna use to calculate the average.

```suggestion
        $countries = Country::withAvg('teams', 'size')->get();
```

Reply:
Funny thing I tried this same function locally and failed the test. But let me try that again, we'll see. Thanks 😊

Reply:
Ok just corrected that now, turns out it works.
I appreciate

Reply:
The reason it worked locally is that your fork wasn't up to date. There was an issue with the `test_countries_with_team_size` where it could be passed by throwing an error. 

This was fixed about 4 days ago after an issue was created that reported it. 🙂